### DISCUSSION TITLE
Hot reload not working inside <q-page-container>

### DISCUSSION THREAD

**Discussion by ReaganM02** [quasarframework/quasar#13792] [General - Components / Directives / etc]
Hi, this is my 10th day of learning Quasar and Vue. While I am building an application that has a dashboard page and a create account page. I noticed that the hot reload is not working inside the `<q-page-container>` tag.  I'm not sure what is going on, I spent 6 hours debugging the issue and I wasn't able to fix it.  The only way to get the updated content inside `<q-page-container>` is when you run `quasar dev` again

Not sure which of these files has an issue but I'll include the possible ones. Since these are the files I recently modified 

routes.js
```
const routes = [
  {
    path: '/',
    name: 'home',
    component: () => import('layouts/MainLayout.vue'),
    children: [
      { path: '', component: () => import('pages/IndexPage.vue') }
    ]
  },
  {
    path: '/dashboard',
    name: 'dashboard',
    component: () => import('layouts/dashboard/dashboardLayout.vue'),
  },

  // Always leave this as last one,
  // but you can also remove it
  {
    path: '/:catchAll(.*)*',
    component: () => import('pages/ErrorNotFound.vue')
  }
]

export default routes
```

Router index.js
```
import { route } from 'quasar/wrappers'
import { createRouter, createMemoryHistory, createWebHistory, createWebHashHistory } from 'vue-router'
import routes from './routes'

/*
 * If not building with SSR mode, you can
 * directly export the Router instantiation;
 *
 * The function below can be async too; either use
 * async/await or return a Promise which resolves
 * with the Router instance.
 */

export default route(function (/* { store, ssrContext } */) {
  const createHistory = process.env.SERVER
    ? createMemoryHistory
    : (process.env.VUE_ROUTER_MODE === 'history' ? createWebHistory : createWebHashHistory)

  const Router = createRouter({
    scrollBehavior: () => ({ left: 0, top: 0 }),
    routes,

    // Leave this as is and make changes in quasar.conf.js instead!
    // quasar.conf.js -> build -> vueRouterMode
    // quasar.conf.js -> build -> publicPath
    history: createHistory(process.env.VUE_ROUTER_BASE)
  })

  return Router
})

```

I also installed the boot file(manually installed) but I deleted the file since it was not working for me. 

Here's the screen record
https://www.youtube.com/watch?v=9W-rpmKBwC0

**Comment by MilosPaunovic** [comment]
Do you get any errors in the browser console?

**Reply by ReaganM02** [reply]
Thanks for replying. No, I don't see any errors or warnings on the console. I think it was fixed last night. What I did is, I created a new project file and copy all the node modules and all the files inside src and install firebase and it is working back to normal.
